jame3-progressing/progress.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
yes, it's random input for now so whenever the real data is changed, it needs to change json file manually. That is the requirement :)